README.md

```diff
@@ -0,0 +1,160 @@
+# rikaikun
+
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
+![Chrome Web Store](https://img.shields.io/chrome-web-store/users/jipdnfibhldikgcjhfnomkfpcebammhp)
+![GitHub](https://img.shields.io/github/license/melink14/rikaikun)
+[![Chrome Web Store](https://img.shields.io/chrome-web-store/v/jipdnfibhldikgcjhfnomkfpcebammhp)](https://chrome.google.com/webstore/detail/rikaikun/jipdnfibhldikgcjhfnomkfpcebammhp)
+[![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg)](https://conventionalcommits.org)
+[![All Contributors](https://img.shields.io/badge/all_contributors-17-orange.svg?style=flat-square)](#contributors-)
+
+rikaikun allows you to read Japanese web pages by hovering over words you don't know and getting their translations.
+
+## Table of Contents
+
+- [rikaikun](#rikaikun)
+  - [Table of Contents](#table-of-contents)
+  - [Background](#background)
+    - [Current Goals](#current-goals)
+  - [Install](#install)
+    - [Via The Store](#via-the-store)
+    - [As an Unpacked Extension](#as-an-unpacked-extension)
+  - [Usage](#usage)
+    - [Features](#features)
+    - [Options](#options)
+      - [Keyboard Shortcuts](#keyboard-shortcuts)
+  - [Maintainers](#maintainers)
+  - [Credits](#credits)
+  - [Contributing](#contributing)
+    - [Contributors ✨](#contributors-)
+  - [License](#license)
+
+## Background
+
+rikaikun is a port of [Rikaichan](https://www.polarcloud.com/getrcx/) I started in 2010 when Chrome released its extension API. It was originally hosted on Google Code until Google Code was canceled and everything was migrated to Github.
+
+For those of you that don't know, Rikaichan was a Firefox extension that emulates the popup translations of rikaixul. It's perhaps the best inline translation of Japanese.
+
+I started with the idea that I would make a the chrome equivalent of Rikaichan from scratch using my own ideas. However, after I tried Rikaichan, I realized that it's already quite good and starting from 0 when he already has numerous iterations would be wasting the excellent work done by those developers.
+
+Since then Firefox killed the extension model Rikaichan was based on and many other variants have been created and ported to various browsers. Many of these were created due to a long period where I left rikaikun stagnant. I apologize to the contributors and users who were left stranded. Thanks for everybody who stayed.
```

Comment:
```suggestion
In 20XX, Firefox killed the extension model that Rikaichan was based on, and other similar extensions have since been created and ported to various browsers.
```

> Many of these were created due to a long period where I left rikaikun stagnant. I apologize to the contributors and users who were left stranded. Thanks for everybody who stayed.

I think it'd be better to matter-of-factly describe the stagnant period, and then focus on a celebratory "We're back!"

Reply:
I focused on the fact that we're back!